mifosng-android/src/main/res/values/strings.xml

```diff
@@ -30,6 +34,7 @@
     <string name="locale">Locale</string>
     <string name="confirm_delete_client_image">Are you sure you want to delete this client\'s image?</string>
     <string name="default_instance_url" translatable="false">demo.openmf.org</string>
+    <string name="url_privacy_policy" translatable="false">http://www.google.com</string>
```

Comment:
@therajanmaurya Please make sure to update it. I added this as demo server was down when I tested.

Reply:
Ok